docs/docs/adding-field-with-default.md

```diff
@@ -43,3 +43,65 @@ ALTER TABLE "core_recipe" ALTER COLUMN "foo" SET NOT NULL;
 We add our column as nullable, set a default for new rows, backfill our column (ideally done in batches to limit locking), and finally remove nullability.
 
 See ["How not valid constraints work"](constraint-missing-not-valid.md#how-not-valid-validate-works) for more information on adding constraints as `NOT VALID`.
+
+
+## solution for alembic and sqlalchemy
+
+Instead of:
+
+```python
+# models.py
+import sqlalchemy as sa
+
+class AlembicValidateMigration(BaseModel):
+    __tablename__ = "alembic_validate_migration"
+    ...
+    some_field_with_non_volatile_default = sa.Column(sa.BigInteger, server_default="10", nullable=False)
```

Comment:
what do you think about us making these smaller column wise / wrapping them more so it's easier to view on mobile and things, like if we run this through black first it gives:


```python
# models.py
import sqlalchemy as sa


class AlembicValidateMigration(BaseModel):
    __tablename__ = "alembic_validate_migration"
    ...
    some_field_with_non_volatile_default = sa.Column(
        sa.BigInteger, server_default="10", nullable=False
    )
```